wattile/models/utils.py

```diff
@@ -18,6 +18,38 @@ def save_model(model, epoch_num, n_iter, filepath):
     )
 
 
+def get_input_dim(configs):
```

Comment:
This code will work for the moment, but if we make the number of extracted features dynamic by passing a list of operators per #204 we will have to revisit. Not a problem, just a note.